### PR TITLE
Additional telemetry in machine configuration

### PR DESCRIPTION
**Pull request by AmelBawa-msft**
## Summary of the pull request
- Additional telemetry in machine configuration

## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: src/appsettings.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "SetupFlowOptions": {
     "StringResourceName": "DevHome.SetupFlow.pri",
     "StringResourcePath": "DevHome.SetupFlow/Resources",
-    "WinGetPackageJsonDataSourcePath": "DevHome.SetupFlow/Assets/AppManagementPackages.json"
+    "WinGetPackageJsonDataSourcePath": "Assets/AppManagementPackages.json"
```
Comment (AmelBawa-msft) [inline src/appsettings.json:9]: Fixing regression after assets were moved

File: tools/SetupFlow/DevHome.SetupFlow/Models/InstallPackageTask.cs [hunk 2]
```diff
@@ -286,7 +288,11 @@ private string GetExtendedErrorCodeMessage()
 
     private void ReportAppSelectedForInstallEvent()
     {
-        TelemetryFactory.Get<ITelemetry>().Log("AppInstall_AppSelected", LogLevel.Critical, new AppInstallUserEvent(_package.Id, _package.CatalogId), _activityId);
```
Comment (AmelBawa-msft) [inline tools/SetupFlow/DevHome.SetupFlow/Models/InstallPackageTask.cs:289]: @RDMacLachlan, is it ok to rename this telemetry event name?
That is emitted when the app install begins. I found the name AppSelected to be a little inaccurate in this case.

**Review by dkbennett — CHANGES_REQUESTED** [review]
General goodness, had some questions in the comments about tagging and the AppInstall_AppSelected rename.

File: tools/SetupFlow/DevHome.SetupFlow/Models/InstallPackageTask.cs [hunk 3]
```diff
@@ -286,7 +288,11 @@ private string GetExtendedErrorCodeMessage()
 
     private void ReportAppSelectedForInstallEvent()
     {
-        TelemetryFactory.Get<ITelemetry>().Log("AppInstall_AppSelected", LogLevel.Critical, new AppInstallUserEvent(_package.Id, _package.CatalogId), _activityId);
+        TelemetryFactory.Get<ITelemetry>().Log("AppInstall_Install", LogLevel.Critical, new AppInstallUserEvent(_package.Id, _package.CatalogId), _activityId);
+        if (_installVersion != _package.DefaultInstallVersion)
+        {
+            TelemetryFactory.Get<ITelemetry>().Log("AppInstall_VersionSpecified", LogLevel.Critical, new AppInstallUserEvent(_package.Id, _package.CatalogId), _activityId);
```
[the remark this answers is not in the document]
Comment (AmelBawa-msft) [reply tools/SetupFlow/DevHome.SetupFlow/Models/InstallPackageTask.cs:294]: Correct

File: tools/SetupFlow/DevHome.SetupFlow/Models/InstallPackageTask.cs (hunk 2 above)
Comment (AmelBawa-msft) [reply tools/SetupFlow/DevHome.SetupFlow/Models/InstallPackageTask.cs:289]: I will keep the original name for now, I know Roy has an email thread about this with the corresponding team. I will reconsider this change separately.

**Review by krschau — APPROVED** [review]

**Comment by AmelBawa-msft** [comment]
@dkbennett _(check comment above)_ 🙂👆

**Pull request merged**